django_mongodb_backend/management/commands/showencryptedfieldsmap.py

```diff
@@ -0,0 +1,47 @@
+from bson import json_util
+from django.apps import apps
+from django.core.management.base import BaseCommand
+from django.db import DEFAULT_DB_ALIAS, connections, router
+
+from django_mongodb_backend.utils import model_has_encrypted_fields
+
+
+class Command(BaseCommand):
+    help = """
+    Shows the mapping of encrypted fields to field attributes, including data
+    type, data keys and query types. The output can be used to set
+    ``encrypted_fields_map`` in ``AutoEncryptionOpts``.
+
+    Defaults to showing keys from the ``key_vault_namespace`` collection.
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--database",
+            default=DEFAULT_DB_ALIAS,
+            help="""
+            Specifies the database to use. Defaults to ``default``.""",
+        )
+        parser.add_argument(
+            "--create-data-keys",
+            action="store_true",
+            help="""
+            If specified, this option will create and show new encryption
+            keys instead of showing existing keys from the configured key vault.
+            """,
+        )
+
+    def handle(self, *args, **options):
+        db = options["database"]
+        create_data_keys = options.get("create_data_keys", False)
+        connection = connections[db]
+        encrypted_fields_map = {}
```

Comment:
```suggestion
        connection = connections[db]
        connection.ensure_connection()
        encrypted_fields_map = {}
```
The command crashes because `connection.connection` is accessed before `connection.database` (similar to https://github.com/mongodb/django-mongodb-backend/pull/412). I don't like having to add `ensure_connection()` but the alternative of `__getattr__()` hacks [5febcd3bcd390a4b52efb0d0f32e176e6334676d removed it] isn't nice either.